.github/workflows/build_linux_pytorch_wheels.yml

```
@@ -3,53 +3,67 @@ name: Build Linux PyTorch Wheels
 on:
   workflow_call:
     inputs:
-      AMDGPU_FAMILIES:
+      amdgpu_family:
         required: true
         type: string
       python_version:
         required: true
         type: string
-      s3_bucket:
+      release_type:
+        description: The type of release to build ("nightly", or "dev")
         required: true
         type: string
       s3_subdir:
+        description: S3 subdirectory, not including the GPU-family
         required: true
         type: string
-      s3_cloudfront:
+      cloudfront_url:
+        description: CloudFront URL pointing to Python index
         required: true
         type: string
+      # rocm_version:
+      #   description: ROCm version to install
+      #   type: string
   workflow_dispatch:
     inputs:
-      AMDGPU_FAMILIES:
+      amdgpu_family:
         required: true
         type: string
       python_version:
         required: true
         type: string
-      s3_bucket:
-        required: true
+        default:
+      release_type:
+        description: The type of release to build ("nightly", or "dev")
         type: string
+        default: "dev"
       s3_subdir:
-        required: true
+        description: S3 subdirectory, not including the GPU-family
         type: string
-      s3_cloudfront:
-        required: true
+        default: "v2"
+      cloudfront_url:
+        description: CloudFront base URL pointing to Python index
         type: string
+        default: "d25kgig7rdsyks.cloudfront.net"
+      # rocm_version:
+      #   description: ROCm version to install (e.g. "==1.0", ">1.0")
+      #   type: string
 
 permissions:
   id-token: write
   contents: read
 
 jobs:
   build_pytorch_wheels:
-    name: Build Linux PyTorch Wheels | ${{ inputs.AMDGPU_FAMILIES }} | Python ${{ inputs.python_version }}
+    name: Build Linux PyTorch Wheels | ${{ inputs.amdgpu_family }} | Python ${{ inputs.python_version }}
     runs-on: ${{ github.repository_owner == 'ROCm' && 'azure-linux-scale-rocm' || 'ubuntu-24.04' }}
     container:
       image: ghcr.io/rocm/therock_build_manylinux_x86_64@sha256:044b113562629f4bd2ec5d2e64b32eee11562d48fb1a75d7493daec9dd8d8292
     env:
       OUTPUT_DIR: ${{ github.workspace }}/output
       PACKAGE_DIST_DIR: ${{ github.workspace }}/output/packages/dist
-      S3_BUCKET_PY: ${{ inputs.s3_bucket }}
+      S3_BUCKET_PY: "therock-${{ inputs.release_type }}-python"
+      ROCM_VERSION: ${{ inputs.rocm_version || '>1.0' }}
```

Should this be commented out also? The input is not declared
```suggestion
      #  ROCM_VERSION: ${{ inputs.rocm_version || '>1.0' }}
```

---

True. Shouldn't hurt as it isn't used but commented it out for consistency. Good catch.